mithril-aggregator/src/database/repository/cardano_transaction_repository.rs

```diff
@@ -242,12 +242,14 @@ impl TransactionStore for CardanoTransactionRepository {
     }
 
     async fn store_transactions(&self, transactions: Vec<CardanoTransaction>) -> StdResult<()> {
+        self.connection.execute("BEGIN TRANSACTION;")?;
```

Comment:
Is it a concern to execute a transaction on very high volume of transactions? Don't we need to do multiple transactions instead?